dispatcher/dispatcher.go

```diff
@@ -397,8 +384,15 @@ func (d *IotxDispatcher) dispatchBlockSyncReq(ctx context.Context, chainID uint3
 		log.L().Debug("no subscriber for this chain id, drop the request", zap.Uint32("chain id", chainID))
 		return
 	}
+	now := time.Now()
+	peerID := peer.ID.Pretty()
 	d.syncChanLock.Lock()
 	defer d.syncChanLock.Unlock()
+	last, ok := d.peerLastSync[peerID]
```

Comment:
I think reserving the subfunc is more readable.
```
if !d.checkSyncPermission(peer.ID.Pretty()) {
return
}
```

Reply:
do you consider this style issue as a blocker of this PR?

Reply:
no